src/lib/api.ts

```diff
@@ -1,15 +1,21 @@
 import { Redis } from '@upstash/redis'
-import { CommonStats } from './types'
+import { CommonStats, ContractStats } from './types'
 import camelcaseKeys from 'camelcase-keys'
 
 const CACHE_EXPIRATION = 3600 * 12
 
 export async function fetchCommonStats(): Promise<CommonStats> {
```

Comment:
No changes required in `fetchCommonStats` function. We should add contract names only in `fetchCommonStatsWithCaching` function

Reply:
done!